addons/l10n_eg_edi_eta/models/account_move.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+import hashlib
+import hmac
+import time
+
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+import logging
+
+import json
+_logger = logging.getLogger(__name__)
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_eg_submission_id = fields.Char(string='Submission ID', compute='compute_eta_response_data', store=True)
+    l10n_eg_uuid = fields.Char(string='Document UUID', compute='compute_eta_response_data', store=True)
+    l10n_eg_signature_data = fields.Text('Signature Data', copy=False)
```

Comment:
not sure we still need this field, because it could be compute store False even.